client/landing/stepper/declarative-flow/internals/steps-repository/site-vertical/index.tsx

```diff
@@ -1,98 +0,0 @@
-import { useIsEnglishLocale } from '@automattic/i18n-utils';
-import { StepContainer } from '@automattic/onboarding';
-import { useDispatch, useSelect } from '@wordpress/data';
-import { useTranslate } from 'i18n-calypso';
-import React from 'react';
-import siteVerticalImage from 'calypso/assets/images/onboarding/site-vertical.svg';
-import DocumentHead from 'calypso/components/data/document-head';
-import FormattedHeader from 'calypso/components/formatted-header';
-import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
-import { useQuery } from '../../../../hooks/use-query';
-import { useSite } from '../../../../hooks/use-site';
-import { SITE_STORE } from '../../../../stores';
-import SiteVerticalForm from './form';
-import type { Step } from '../../types';
-import type { SiteSelect } from '@automattic/data-stores';
-import type { Vertical } from 'calypso/components/select-vertical/types';
-
-const SiteVertical: Step = function SiteVertical( { navigation } ) {
-	const { goBack, goNext, submit } = navigation;
-	const [ vertical, setVertical ] = React.useState< Vertical | null >();
-	const [ isBusy, setIsBusy ] = React.useState( false );
-	const { saveSiteSettings } = useDispatch( SITE_STORE );
-	const site = useSite();
-	const siteVertical = useSelect(
-		( select ) =>
-			( site && ( select( SITE_STORE ) as SiteSelect ).getSiteVerticalId( site?.ID ) ) || undefined,
```

Comment:
I wonder if we can remove [getSiteVerticalId](https://github.com/Automattic/wp-calypso/blob/4e51b4833dfe46b3112cac0625cfa8571c851976/packages/data-stores/src/site/selectors.ts#L46) because it's only used in this file and we are removing it.

Reply:
Sure! I thought we could leave it for parity since we are keeping `setSiteVerticalId`, but it should be easy to add it back if needed.